src/styles/components/_usage-guidelines.scss

```diff
@@ -1,55 +1,59 @@
-$breakpoint-tablet: 1220px;
+$breakpoint-tablet: 860px;
 
 .cmp-usage-guidelines {
-  background-color: #01f5ac;
-  padding-left: 1.25rem;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  line-height: 1.875rem;
+  letter-spacing: 0.02rem;
 
   &__heading {
     position: relative;
     font-family: Viga, sans-serif;
     font-weight: 400;
     font-size: map.get($font-sizes, "md");
-    padding-top: 3.75rem;
+    margin-top: 0;
   }
 
-  &__content {
-    padding-right: 1.25rem;
-    padding-bottom: 1.25rem;
+  &__image {
+    position: relative;
+    width: 80%;
+    max-width: 30rem;
```

Comment:
I think this looks a lot less huge/better than it did before with its size capped off, nice work! 🎉 

<img width="419" alt="image" src="https://user-images.githubusercontent.com/54716846/200096934-286f9b81-1202-43ee-a42b-9f56738cd40a.png">